modules/refined/src/test/scala/doobie/refined/RefinedSuite.scala

```diff
@@ -80,26 +78,26 @@ class RefinedSuite extends munit.FunSuite {
       _ <- Update0(s"CREATE LOCAL TEMPORARY TABLE TEST (value INT)", None).run
       _ <- sql"INSERT INTO TEST VALUES ($v)".update.run
     } yield ()
-    queryRes.transact(xa).unsafeRunSync()
+    queryRes.transact(xa).assert
   }
 
   test("Query should throw an SecondaryValidationFailed if value does not fit the refinement-type ") {
     secondaryValidationFailedCaught_?(
-      sql"select -1".query[PositiveInt].unique.transact(xa).void.unsafeRunSync()
+      sql"select -1".query[PositiveInt].unique.transact(xa).void.assert
```

Comment:
`.assert` seems redundant here and below – the exception is going to be intercepted inside `secondaryValidationFailedCaught_?`.